src/seedu/addressbook/data/person/Person.java

```diff
@@ -78,10 +78,19 @@ public int hashCode() {
         // use this method for custom fields hashing instead of implementing your own
         return Objects.hash(name, phone, email, address, tags);
     }
-
+    
+    
     @Override
     public String toString() {
         return getAsTextShowAll();
     }
+    
+    public static String getPrintableString(Printable... printables) {
+        String s = ""; 
+        for (Printable p : printables) {
+            s += p.getPrintableString() + ", ";
```

Comment:
Doesn't this result in an extra `,` at the end? You could have used `StringJoiner` or `String.join` instead.